docs/references/expo/use-sso.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: '`useSSO()`'
+description: Clerk's useSSO() hook is used to create a new SSO flow.
+---
+
+The `useSSO()` hook is used to create a new SSO flow. It can be used in both web and native apps.
+
+## Parameters
+
+<Properties>
+  - `strategy`
+
+  - [`OAuthStrategy`](/docs/references/javascript/types/sso#o-auth-strategy)
+
+  - [`EnterpriseSSOStrategy`](/docs/references/javascript/types/sso#enterprise-sso-strategy)
+
+  - `strategy?`
```

Comment:
Just confirming, is this `strategy` under `EnterpriseSSOStrategy`? I think the formatting got a little odd.

Reply:
I've rollback the `useSSO` file from this PR, it was accidentally added as I mentioned [here](https://github.com/clerk/clerk-docs/pull/1907#discussion_r1920568597)